gateway/requests/requests.go

```diff
@@ -90,3 +90,20 @@ type AsyncReport struct {
 type DeleteFunctionRequest struct {
 	FunctionName string `json:"functionName"`
 }
+
+// CreateSecretRequest create a secret w/ annotations
+type CreateSecretRequest struct {
```

Comment:
Do we need a ListSecret equivalent or will that just be `[]Secret`?

Reply:
I was just thinking []Secret as the response type for GET.